src/app/pages/partners/partner-details/synopsis/synopsis.js

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect, useRef} from 'react';
+import StarsAndCount from '~/components/stars-and-count/stars-and-count';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import './synopsis.css';
+
+function VerifiedBadge({verifiedFeatures}) {
+    const badgeImage = '/images/partners/verified-badge.svg';
+
+    return (
+        verifiedFeatures &&
+            <div className="badge">
+                <img className="background" src={badgeImage} alt="verified" />
+                <FontAwesomeIcon icon="check" className="checkmark" />
+                <div className="tooltip bottom">
+                    {verifiedFeatures}
+                </div>
+            </div>
+    );
+}
+
+function logScrollingInParent(el, name) {
+    const scrollingRegion = el.parentNode.parentNode;
+    // eslint-disable-next-line prefer-const
+    let scrollCallback;
+    const removeScrollListener = () => scrollingRegion.removeEventListener('scroll', scrollCallback);
```

Comment:
maybe I'm missing it but I don't see `scrollCallback` being reassigned

Reply:
This is an awkward construct due to mutual dependency between scrollCallback and removeScrollListener, which makes lint unhappy about "used before defined". I'd welcome a more elegant solution if one suggests itself to you.